libr/include/experimental/x_m1.h

```diff
@@ -0,0 +1,26 @@
+#ifndef X_M1_H
+#define X_M1_H
+
+#include <r_bin.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+//R_API void r_bin_sorted_section_free (void  /*RBinSortedSection*/ *data);
+//R_API int r_bin_sorted_section_vaddr_cmp (const RBinSortedSection *a, const RBinSortedSection *b);
+//R_API int r_bin_sorted_section_paddr_cmp (const RBinSortedSection *a, const RBinSortedSection *b);
+//R_API int r_bin_sorted_section_contains_addr (const RBinSortedSection *a, ut64 off, int va);
+R_API ut64 r_bin_section_get_from_addr (RBinObject *o, RBinSection *s, int va);
+R_API ut64 r_bin_section_get_to_addr (RBinObject *o, RBinSection *s, int va);
```

Comment:
this thing has been solved in siol with a binary tree, not sure if that would be faster here instead of adding an experimental api